src/main/java/org/spongepowered/api/channel/Channel.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.channel;
+
+import java.util.Collection;
+
+import javax.annotation.Nullable;
+
+/**
+ * @param <T> The type to be sent
+ * @param <M> The member type
+ */
+public interface Channel<T, M> {
+
+    /**
+     * Sends a {@link T} to this channel's members.
+     *
+     * @param original The original text to send
+     */
+    default void send(T original) {
+        this.send(null, original);
+    }
+
+    /**
+     * Sends a {@link T} to this channel's members.
+     *
+     * @param sender The sender of the message
+     * @param original The original message to send
+     */
+    void send(@Nullable Object sender, T original);
```

Comment:
All situations where the targets aren't known.

* Async/delayed message delivery. (wait for the end of the match/turn to deliver the message)
* Network based communication (post the message in a forum/thread)
* Log messages to file

Situations where you want to send something but you simply don't/shouldn't care who will get the message.
(Similar to sending a network package to a broadcast IP. You don't care whether someone is listening or who gets the message you just want to send it.)